docker/nextstrain-errorhandler/Dockerfile

```diff
@@ -0,0 +1,30 @@
+## Dockerfile for aspen batch jobs
+##
+##
+FROM nextstrain/base
+ARG REVSPEC=trunk
+ARG ASPEN_DOCKER_IMAGE_VERSION
+ENV ASPEN_DOCKER_IMAGE_VERSION=${ASPEN_DOCKER_IMAGE_VERSION}
+
+RUN apt-get install -y git
+
+WORKDIR /aspen
+
+# Prepopulate the virtual environment by installing the dependencies of aspen as of a fixed rev.
+# Assuming the dependencies do not change from the time the docker image is built to the time the
+# docker image is used, this cuts startup time for the docker image.  If the dependencies DO change,
+# then pip will uninstall the prepopulated dependencies and install the correct dependencies.
+RUN mkdir /tmp/aspen-tmp
+RUN cd /tmp/aspen-tmp
+RUN git init
+RUN git fetch git://github.com/chanzuckerberg/aspen $REVSPEC
+RUN git checkout FETCH_HEAD
+RUN python3.7 -m venv /aspen/.venv
```

Comment:
this is using python3.7 but everything else has moved to 3.9

Reply:
It's just a copy of the nextstrain dockerfile for the moment. More work TBD here.